Zyantific.Zydis/Native/DecoderTypes.cs

```diff
@@ -1,9 +1,26 @@
 ﻿using System.Runtime.InteropServices;
 
 using ZyanU8 = System.Byte;
```

Comment:
I'm still not sure if we should define the types in every header, but we can leave it like this for now.